docker-compose.template

```diff
@@ -36,17 +38,17 @@ hydroshare:
     SOLR_HOST: solr
   volumes:
     # hydroshare repository
-    - "HS_PATH:/home/docker/hydroshare"
+    - "HS_PATH:/hydroshare"
     # pycharm debugging
-    # - "/home/hydro/pycharm-debug:/home/docker/pycharm-debug"
+    # - "/home/hydro/pycharm-debug:/pycharm-debug"
     # log files
-    - "HS_LOG_FILES:/var/log/hydroshare"
+    - "HS_LOG_FILES:/hydroshare/log"
     # shared location for uwsgi.sock between containers
-    - "/var/run"
+    - "/hs_tmp"
     # temp directory shared with celery workers
-    - "/shared_temp"
+    - "/shared_tmp"
   ports:
-    - "1338:22"
+    - "1338:2022"
```

Comment:
Want the ssh service to be run/owned by the hydro-service account, not root, so using a non-standard port, sshd will not be started on port 22 at all.
This will run as long as USE_SSL is not set to true.

Will require an update to the deployment configuration in PyCharm for remote SFTP for the new service account. But otherwise should be transparent when using ssh from host (user/pass = hydro-service/docker)
For example: `ssh -p 1338 hydro-service@192.168.56.101`